bank.rb

```diff
@@ -0,0 +1,50 @@
+$balance = 0
```

Comment:
Looks good, but let's encapsulate this behavior in a Class. Try rewriting this using ```Class Bank``` and defining methods within it such as deposit, withdraw, etc.